experimental/dds/tree2/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -799,6 +736,74 @@ export class ModularChangeFamily
 	}
 }
 
+export function intoDelta(
```

Comment:
I am legally required to inform you that our official FF policy requires documentation on module-exported functions.

Reply:
perfect, I have just the comment to reuse